clang/include/clang/Frontend/FrontendOptions.h

```diff
@@ -444,6 +444,10 @@ class FrontendOptions {
   // Enable Clang IR library optimizations
   unsigned ClangIRLibOpt : 1;
 
+  // Enable Clang IR analysis only pipeline that uses tranditional code gen
+  // pipeline.
+  unsigned ClangIRAnalysisOnlyPipeline : 1;
```

Comment:
Same for all related names, you can cut the pipeline and just use `ClangIRAnalysisOnly`

Reply:
Done